jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -638,6 +639,29 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         managerState.flags.putAll(state.flags)
     }
 
+    private fun findBuildFiles(sourceFolder: File): List<File> {
```

Comment:
Question: Do we have a test to make sure this doesn't find the file after N depth?

Would this be too much effort to create and not a lot of value proposition in that test?

Reply:
Not much value imo, more importantly is that we have tests for nested and poms not in the root which we both have.